src/transformers/hf_argparser.py

```diff
@@ -25,6 +25,18 @@
 DataClassType = NewType("DataClassType", Any)
 
 
+# From https://stackoverflow.com/questions/15008758/parsing-boolean-values-with-argparse
+def string_to_bool(v):
+    if isinstance(v, bool):
+        return v
+    if v.lower() in ("yes", "true", "t", "y", "1"):
+        return True
+    elif v.lower() in ("no", "false", "f", "n", "0"):
+        return False
+    else:
+        raise ArgumentTypeError("Boolean value expected.")
```

Comment:
(nit) would it make more sense to say `"Truthy value expected that  is either of type `bool` or one of ['yes', ....]"`